src/ex1/index.html

```diff
@@ -3,12 +3,29 @@
 <head>
     <title>Exercise 1</title>
     <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/7a715c599c.js" crossorigin="anonymous"></script>
+
 </head>
 <body>
+    <div class="background">
+        <div class="main">
+            <h1>To-Do List</h1>
+            <div class="add-task">
+                <input type="text" name="new-task" value="" placeholder="Add your new todo" class="add-task-field" />
+                <span class="add-task-btn">
+                    <i class="fa-solid fa-square-plus fa-2x"></i>
```

Comment:
also just a syntax suger, you can use self closing elements
```suggestion
                    <i class="fa-solid fa-square-plus fa-2x" />
```

Reply:
That destroys the page for some reason and the -> / highlighted in red.
Any ideas why?